scapy/tools/automotive/xcpscanner.py

```diff
@@ -0,0 +1,167 @@
+#! /usr/bin/env python
+
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Fabian Wiche <f.wiche@gmx.de>
```

Comment:
@TabeaSpahn you should put your name here

Reply:
I actually did not write this code. Fabian did. I just saw that after I opened the PR.
So I have to check this first, and then I can add my name.